envsec/internal/envcli/flags.go

```diff
@@ -26,12 +26,12 @@ func (f *configFlags) register(cmd *cobra.Command) {
 		"Project id to namespace secrets by",
 	)
 
-	cmd.PersistentFlags().StringVar(
-		&f.orgId,
-		"org-id",
-		"",
-		"Organization id to namespace secrets by",
-	)
+	// cmd.PersistentFlags().StringVar(
+	// 	&f.orgId,
+	// 	"org-id",
+	// 	"",
+	// 	"Organization id to namespace secrets by",
+	// )
```

Comment:
Can we make this change in a way that doesn't break existing open source functionality?